### ISSUE TITLE
[Masthead]: A11y - Hyperlink has no link text, label or image with a text alternative

### ISSUE THREAD

**Issue by rohitrajmishra** [carbon-design-system/carbon-for-ibm-dotcom#8345]
### Description

===== DEFECT DESCRIPTION =====
Hyperlinks must have a text description of their purpose. When the purpose of a link is clear users can easily navigate links on the page without having to see the surrounding information for context.

### Component(s) impacted

Masthead

### Browser

Chrome, Safari, Firefox, Microsoft Edge

### Carbon for IBM.com version

v1.11.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

cloud.ibm.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://codesandbox.io/s/github/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/react/examples/codesandbox/components/Masthead?file=/src/App.js

### Steps to reproduce the issue (if applicable)

===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: 2.4.4 Link Purpose (in Context)
IBM Guidance: https://www.ibm.com/able/requirements/requirements/?version=7_1#2_4_4
Test Tool Type: IBM Equal Access Accessibility Checker

=====STEPS TO REPRODUCE =====

1. Open https://dev.console.test.cloud.ibm.com/docs/discovery-data
2. Check Accessibility

===== EXPECTED RESULT =====
- Ideally, add unique link text as the content of the hyperlink that describes the purpose of the hyperlink;

- Or, if it’s not possible to use unique link text (e.g. multiple ‘Read more..’ links for different topics), use the aria-labelledby attribute to point to text visible on the page that identifies the link purpose. The following example shows how the aria-labelledby provides additional context by adding the text 'Breaking News' from the header to the link text 'Read more..':

<img width="1437" alt="image" src="https://media.github.ibm.com/user/140782/files/14e3c480-4d4b-11ec-95cf-aa6981a860f3">

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by annawen1** [comment]
Running the checker on the latest version and I see those errors coming from the checker on the Masthead Profile Items. However, looking at the code, it seems the checker hasn't been adopted for web-components as our link text is passed in via slots.

<img width="872" alt="Screen Shot 2022-03-15 at 2 08 58 PM" src="https://user-images.githubusercontent.com/54281166/158443989-d60972b7-eff9-4030-9c53-c5b31320a332.png">

<img width="915" alt="Screen Shot 2022-03-15 at 2 09 42 PM" src="https://user-images.githubusercontent.com/54281166/158444005-4c36b8bd-fb8a-42c0-9438-3b28dace27fe.png">

**Comment by annawen1** [comment]
@ariellalgilmore tagging you in this ticket as you're reaching out to the accessibility team about this issue!

**Comment by ariellalgilmore** [comment]
issue opened in the equal access toolkit: https://github.com/IBMa/equal-access/issues/741

**Comment by proeung** [comment]
@annawen1 @ariellalgilmore Thanks for following up on this issue. Since you both have already opened issues to the IBM accessibility team, I'll take our team out of this issue for now and move this back to "Blocked from Scheduling" until we hear back from the Equal Access group.

**Comment by rohitrajmishra** [comment]
Hi @annawen1 @ariellalgilmore @proeung ,

Is there any update on this issue?

I checked that the issue is closed/fixed in equal access toolkit: 
https://github.com/IBMa/equal-access/issues/741 and https://github.com/IBMa/equal-access/issues/743 

Are these changes integrated in carbon-for-ibm-dotcom?
If yes, please let us know the relevant version.

**Comment by andy-blum** [comment]
This issue appears to no longer exist. The accessibility checker on cloud console no longer shows this issue as a violation.

<img width="1440" alt="Screenshot 2022-12-22 at 12 38 15 PM" src="https://user-images.githubusercontent.com/25532785/209193926-32108baa-c72a-464c-b928-5973ffe91c9d.png">
